src/main/java/at/hannibal2/skyhanni/features/garden/contest/JacobFarmingContestsInventory.kt

```diff
@@ -73,6 +74,66 @@ class JacobFarmingContestsInventory {
         realTime[slot] = "$dayFormat $startTimeFormat-$endTimeFormat"
     }
 
+    @SubscribeEvent
+    fun onSlotClick(event: SlotClickEvent) {
+        if (!config.openOnline.isKeyHeld()) return
+        if (!LorenzUtils.inSkyBlock) return
+        val invName = InventoryUtils.openInventoryName()
+        val name = event.slot.stack.name
+
+        when (invName) {
+            "Your Contests" -> {
+                FarmingContestAPI.timePattern.matchMatcher(name) {
+                    val date = getDate(group("year"), group("month"), group("day"))
+                    OSUtils.openBrowser("https://elitebot.dev/contests/$date")
+                    ChatUtils.chat("Opening contest in elitebot.dev")
```

Comment:
Those three lines are duplicated a bit below. Create a function `openContestWebsite(year, month, day)` instead and use it twice